dwitter/static/js/ajax-handling.js

```diff
@@ -25,54 +24,59 @@ var processLike = function(e) {
     },
     success: processServerResponse,
   };
+
+  e.preventDefault();
   $.ajax(config);
-};
+}
 
-var getCommentHTML = function(comment) {
+function getCommentHTML(comment) {
   return '<li class=comment><a class=comment-name href="/u/' + comment.author + '">' +
     comment.author + ':</a> ' +
     '<span class="comment-message">' + comment.urlized_text +
     '</span></li>';
-};
+}
 
-var loadComments = function() {
+function loadComments() {
   var step = 1000;
   var currentOffset = $(this).data('offset');
 
   var $loadCommentsButton = $(this);
 
   var dweetId = $loadCommentsButton.data('dweet_id');
   // If there is a sticky comment on the top of the comments
-  var stickyTop = $loadCommentsButton.data('stickyTop');
+  var stickyTop = $loadCommentsButton.data('sticky_top');
```

Comment:
I believe this fixes a bug where the wrong data attribute was being read off of the button. 😨